lessons/module-1/return-and-srp.md

```diff
@@ -56,9 +56,9 @@ We can use this to our advantage when trying to stop the execution of a program
 
 These examples may be familiar, but they become more important when considering how to make our functions adhere to the single responsibility principle. 
 
-### Single Responsibilty Principle
+### Single Responsibility Principle
 
-The single responsibility principle is the first principle of object oriented design. The term was coined by Robert C. Martin, a computer programmer and co-author of the Agile Manifesto. It states that 
+The single eesponsibility principle is the first principle of object oriented design. The term was coined by Robert C. Martin, a computer programmer and co-author of the Agile Manifesto. It states that 
```

Comment:
@nicktu12 - Thanks for reviewing the lessons and fixing the typos, I noticed a small typo: 'eesponsibility' should be 'responsibility'.